lib/portlayer/exec/exec.go

```diff
@@ -152,8 +158,36 @@ func eventCallback(ie events.Event) {
 
 			}
 		}
+		return
 	}
+	// check container registered event if this container is not found in container cache
+	switch ie.String() {
+	case events.ContainerRegistered:
+		moref := new(types.ManagedObjectReference)
+		if ok := moref.FromString(ie.Reference()); !ok {
+			log.Errorf("Failed to get event VM mobref: %s", ie.Reference())
+			return
+		}
+		var vm mo.VirtualMachine
+
+		// current attributes we care about
+		attrib := []string{"config", "runtime.powerState", "summary", "resourcePool"}
```

Comment:
I knew that. But I need one more attribute for the resource pool. If call populateVMAttributes, I'll need to call property collector twice.